src/main/java/frc/robot/subsystems/flywheel/flyIONonVariable.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.subsystems.flywheel;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.DemandType;
+import com.ctre.phoenix.motorcontrol.InvertType;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.StatorCurrentLimitConfiguration;
+import com.ctre.phoenix.motorcontrol.SupplyCurrentLimitConfiguration;
+// External Imports
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+import edu.wpi.first.math.util.Units;
+// Internal Imports
+import frc.robot.Constants;
+
+public class flyIONonVariable implements flyIO {
```

Comment:
```suggestion
public class FlywheelIONonVariable implements FlywheelIO {
```

Reply:
Classes and interfaces should have uppercase names (pascal case)

Reply:
What is meant by NonVariable?

Reply:
ah we were planning on having a variable hood shooter some time before dchamps, just setting up the infastructure so that we can change between variable and non variable hoods

Reply:
Ah okay, I see